workshops/erc20-mts/scripts/cli-denom.sh

```
@@ -0,0 +1,26 @@
+
+source .local.env
+
+# Check if an argument was provided
+if [ -z "$1" ]; then
+  echo "⚠️  Error: No denom specified"
+  echo "Usage: $0 <denom>"
+  exit 1
+fi
+
+# Check if denom starts with erc20: prefix
+if [[ "$1" != erc20:* ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)